backend/tests/FuzzTests/Generators.fs

```diff
@@ -192,16 +192,32 @@ module RuntimeTypes =
                       //(4, ok) // TODO: random string, with 0-5 args
                        ]
 
-  open MatchPattern
+    let rec genTuple (s, genArg) : Gen<RT.MatchPattern> =
```

Comment:
Looks like I backfilled this, but didn't 'use' it below. Not top-priority but might as well use it below in the file